tests/tests.py

```diff
@@ -0,0 +1,38 @@
+from tweetcloud import helpers
+
+
+def test_normalize_and_split_tweet():
```

Comment:
if one assert fails it'll skip checking later ones. Might be useful to have everything separate functions, or parameterized tests https://docs.pytest.org/en/latest/example/parametrize.html

Not really necessary to do now just something to think about